frontend/packages/kubevirt-plugin/src/k8s/requests/vm/index.ts

```diff
@@ -0,0 +1,48 @@
+import { k8sPatch, resourceURL } from '@console/internal/module/k8s';
+import { getName, getNamespace } from '@console/shared';
+import { coFetch } from '@console/internal/co-fetch';
+import { analyticsSvc } from '@console/internal/module/analytics';
+import { getPxeBootPatch } from '../../patches/vm/vm-boot-patches';
+import { VirtualMachineModel } from '../../../models';
+import { VMKind } from '../../../types/vm';
+
+export enum VMActionEnum {
```

Comment:
I wouldn't suggest using `Enum` suffix for enum types, can't we simply have `VMActionType`?

I also see `packages/kubevirt-plugin/src/components/vm-disks/types.ts` has `StorageTypeEnum`, can we update that one to `StorageType` as well?